MINOR-INSTALLATION.md

```diff
@@ -0,0 +1,63 @@
+# Minimal Installation
+
+#### Build Dependencies
+* gcc (9.3.0-17ubuntu1~20.04)
+* make (4.2.1-1.2)
+* libtool (2.4.6-14)
+* curl (7.68.0-1ubuntu2.5)
+* cmake (3.16.3-1ubuntu1)
+* libssl-dev (1.1.1f-1ubuntu2.4)
+* liburiparser1 (0.9.3-2)
+* liburiparser-dev (0.9.3-2)
+* protobuf-compiler (3.13.0)
+* libprotobuf-dev (3.13.0)
+* libjansson-dev (2.12-1build1)
+* libcjose-dev (0.6.1+dfsg1-1)
+* libcurl4-openssl-dev (7.68.0-1ubuntu2.5)
+* check (0.10.0-3build2)
+
```

Comment:
```suggestion
Install dependencies with `apt-get update && apt-get install -y build-essential automake libtool curl cmake libssl-dev liburiparser1 liburiparser-dev protobuf-compiler libprotobuf-dev libjansson-dev libcjose-dev libcurl4-openssl-dev check`
```
Add this from the PR's description, since installing them separately as stated above didn't get me all dependencies.